src/OpenTracing/GlobalTracer.php

```diff
@@ -18,7 +18,7 @@
      *
      * @param Tracer $tracer
      */
-    public static function setGlobalTracer(Tracer $tracer)
+    public static function withGlobalTracer(Tracer $tracer)
```

Comment:
I think there are meaningful ways in most of the cases to express the intention behind it. For example, `setStatus('active')` can be better described by `activate()` which is the actual use case. I don't believe an object should be told `setYourStateTo`, I'd rather prefer the object itself knowing how to change it's own state by `doSomething()`